gcdm/build.gradle.kts

```diff
@@ -0,0 +1,112 @@
+import com.google.protobuf.gradle.generateProtoTasks
+import com.google.protobuf.gradle.id
+import com.google.protobuf.gradle.plugins
+import com.google.protobuf.gradle.protobuf
+import com.google.protobuf.gradle.protoc
+
+// bug in IntelliJ in which `libs` shows up as not being accessible
+// see https://youtrack.jetbrains.com/issue/KTIJ-19369
+@Suppress("DSL_SCOPE_VIOLATION")
+
+plugins {
+    id("cdm.library-conventions")
+    id("application")
+    alias(libs.plugins.protobufPlugin)
+    alias(libs.plugins.execforkPlugin)
+}
+
+// Matches Maven's "project.description"
+description = "gRPC client and server implementation of CDM Remote Procedure Calls (gCDM)."
+
+dependencies {
+    implementation(platform(project(":netcdf-java-platform")))
+    implementation(platform(libs.grpcBom))
```

Comment:
what is this?

Reply:
what is this Bom thing?

Reply:
The bom lists all of the grpc projects and their versions in one file:

https://repo1.maven.org/maven2/io/grpc/grpc-bom/1.43.1/grpc-bom-1.43.1.pom

Two nice things about using it. 1. You only specify the version once, and anything you reference from the bom picks up the version from the bom, and 2. using the bom as a gradle "platform" will also handle aligning the versions of transitive dependencies.